libs/blocks/global-footer/global-footer.js

```diff
@@ -114,20 +122,25 @@ class Footer {
     try {
       return new DOMParser().parseFromString(parsedHTML, 'text/html').body;
     } catch (e) {
-      // TODO: log to LANA if Footer could not be instantiated
+      lanaLog({ message: 'Footer could not be instantiated', tags: 'errorType=error,module=global-footer' });
       return null;
     }
   };
 
   loadMenuLogic = async () => {
```

Comment:
This should bubble up all the way to `decorateContent`if an error is happening here, any reason to specifically add a try/catch here?
`decorateContent (try/catch/logErrorFor)` ->  `decorateGrid` -> `loadMenuLogic`

Reply:
Right, the error will bubble up so no reason to have a try/catch in there